packages/libs/react-ui/src/components/InputWrapper/InputWrapper.tsx

```diff
@@ -13,13 +14,15 @@ export interface IInputWrapperProps extends Omit<IInputHeaderProps, 'label'> {
   disabled?: boolean;
   helperText?: string;
   label?: string;
+  labelWidth?: keyof typeof vars.sizes;
```

Comment:
the name is a bit misleading. I think this should be "leadingTextWidth" or something since the label for this component is referring to something else